ui/k-toolbar/main.kytos

```diff
@@ -12,6 +12,7 @@
                     placeholder="First endpoint" icon="arrow-right"
                     :candidates="dpids" 
                     @focus="fetchDpids"
+                    @blur="onblur_dpid"
```

Comment:
@rmotitsuki, when I was trying this PR out, the `candidates` list didn't show up in the component, do I have to update the UI package or something else to see this in action? really great to have these candidates for auto completion
![2022-01-10-145154_1919x504_scrot](https://user-images.githubusercontent.com/1010796/148815214-27debd7d-9471-42c3-9b3c-a5cf73e8e12e.png)

Reply:
After updating the ui to `1.4.3-beta.2` it worked, thanks @rmotitsuki, I took the opportunity to also create and delete an EVC via the UI. 

![2022-01-10-170609_892x803_scrot](https://user-images.githubusercontent.com/1010796/148832229-3abd01fa-ba8e-4ebd-aa88-7c927d1e3a7c.png)